src/lib/certmap/sss_certmap_ldap_mapping.c

```diff
@@ -189,10 +461,17 @@ static int parse_template(TALLOC_CTX *mem_ctx, struct sss_certmap_ctx *ctx,
         goto done;
     }
 
-    ret = check_parsed_template(ctx, parsed);
+    /* If the template cannot be found in the base table,
+     * check LDAPU1 as well. */
+    ret = check_parsed_template(ctx, template_table_base, parsed);
```

Comment:
It's required to check against base table first because LDAPU1 is a superset of LDAP type?

Reply:
Yes.